ci/scripts/gitlab/common.sh

```diff
@@ -36,23 +36,28 @@ function get_git_tag() {
 
         # If the branch is a nightly build create a version which will be accepted by pypi
         # Note: We are intentionally creating an actual tag, just setting the variable
-        GIT_TAG=$(echo $GIT_TAG | sed -e "s|-.*|a$(date +"%Y%m%d")|")
+        GIT_TAG=$(echo $GIT_TAG | sed -E -e "s/(-|a).*/a$(date +"%Y%m%d")/")
```

Comment:
Is this just replacing all of the cruft after `-` or `a` with `aYYYYMMDD` ? Maybe a comment above would be nice.

Also, i think the note above is missing a **not**